### ISSUE TITLE
Incorrect Unirule mapping (UR001419768)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#3995]
UniProt UniRule (ID and label):
UniRule: UR001419768

Sequences with problematic annotation (ID + gene/protein name):
fly Polr3C (Q9VG60)

Description of issue
UR001419768 annotates 'DNA-directed RNA polymerase III subunit RPC3' with "GO:0003899 DNA-directed 5'-3' RNA polymerase activity" and the keyword "DNA-directed RNA polymerase" (which also generates a "DNA-directed 5'-3' RNA polymerase activity" annotation).  But this subunit does not form part of the catalytic core of RNAP3 (e.g. see PMID: 22365827).  Can that GO term and KW be removed from this UniRule please?

**Comment by pgaudet** [comment]
@Antonialock are you handling these?

**Comment by Antonialock** [comment]
@pgaudet I would like to assign these to Pedro and Elena. I can't figure out how.....they are requesting membership now to see if that makes it possible.

**Comment by pgaudet** [comment]
(note that I have added a new template for UniRules with the correct contact persons)

**Comment by Antonialock** [comment]
Super, thanks

**Comment by sjm41** [comment]
Thanks both.  And a new label too :)

**Comment by sjm41** [comment]
_We agree that the subunit does not have a catalytic activity so we changed the GO term "GO:0003899 DNA-directed 5'-3' RNA polymerase activity" to another one "GO:0000428 DNA-directed RNA polymerase complex". This is because we still want to characterize this protein as part of the complex, but that it does not have a catalytic activity._

Sounds good, thanks!

_In regards of the KW "DNA-directed RNA polymerase", we kept this as we mean to communicate to the user that these proteins appear to form a subcomplex within DNA-directed RNA polymerase complex._

Understood.  But unless the KW mapping from "DNA-directed RNA polymerase" to "DNA-directed 5'-3' RNA polymerase activity GO:0003899 (https://www.uniprot.org/keywords/KW-0240)" is removed, we'll still have the same problem! (i.e. Non-catalytic proteins like Q9VG60 will still end up getting "DNA-directed 5'-3' RNA polymerase activity [ GO:0003899 ]")
Can that KW2GO association be removed?
